src/Hook/EnableDisable.php

```diff
@@ -50,6 +50,11 @@ private function onBeforeAll(BeforeAll $event)
         $this->bag->add(
             $this->hookHandler->getJsonHookDataWithDefaults(static::class, $defaults)
         );
+
+        if (count($this->bag->get("disable")) > 0) {
```

Comment:
Let's replace the double quotes with single ones `if (count($this->bag->get('disable')) > 0) {`